repo/packages/Q/quobyte/0/marathon.json

```diff
@@ -0,0 +1,42 @@
+{
+  "id": {{quobyte.framework-name}},
+  "cpus": "1.0",
+  "mem": "1024",
+  "instances": "1",
```

Comment:
all resources and instances need to be configurable.  cpu, mem and instances need config values from config.json